libimage/image_tree.go

```diff
@@ -37,7 +38,7 @@ func (i *Image) Tree(traverseChildren bool) (string, error) {
 		fmt.Fprintf(sb, "No Image Layers")
 	}
 
-	layerTree, err := i.runtime.layerTree(nil)
```

Comment:
Wouldn't it make sense to force users of libimage to pass in a context

Reply:
That would be breaking API change, and so far I do not see the need of the context so this would juts make our live harder when vendoring into podman/buildah.